lib/katello/tasks/upgrade_check.rake

```diff
@@ -0,0 +1,36 @@
+DESIRED_VERSION = "2.4.2"
+SYSTEM = "Katello"
```

Comment:
Is the idea Satellite could override this? We're not consistently using it then, e.g. Katello 3.0 appears in the strings below